perun-ldapc-ada/src/main/resources/perun-ldapc.xml

```diff
@@ -272,6 +272,36 @@ http://www.springframework.org/schema/aop http://www.springframework.org/schema/
 					<property name="pattern" value=" added to Group:\[(.|\s)*\]" />
 					<property name="handlerMethodName" value="processMemberAdded" />
 				</bean>
+				<bean class="cz.metacentrum.perun.ldapc.processor.impl.RegexpDispatchEventCondition">
+					<property name="beansCondition">
+						<list>
+							<value>cz.metacentrum.perun.core.api.Group</value>
+							<value>cz.metacentrum.perun.core.api.Member</value>
+						</list>
+					</property>
+					<property name="pattern" value="  validated.$" />
+					<property name="handlerMethodName" value="processMemberAdded" />
+				</bean>
+				<bean class="cz.metacentrum.perun.ldapc.processor.impl.RegexpDispatchEventCondition">
+					<property name="beansCondition">
+						<list>
+							<value>cz.metacentrum.perun.core.api.Group</value>
+							<value>cz.metacentrum.perun.core.api.Member</value>
+						</list>
+					</property>
+					<property name="pattern" value=" was removed from Group:\[(.*)\] totally" />
+					<property name="handlerMethodName" value="processMemberRemoved" />
+				</bean>
+				<bean class="cz.metacentrum.perun.ldapc.processor.impl.RegexpDispatchEventCondition">
+					<property name="beansCondition">
+						<list>
+							<value>cz.metacentrum.perun.core.api.Group</value>
+							<value>cz.metacentrum.perun.core.api.Member</value>
+						</list>
+					</property>
+					<property name="pattern" value="expired.$|disabled.$|invalidated.$|suspended #" />
+					<property name="handlerMethodName" value="processMemberRemoved" />
+				</bean>
```

Comment:
These regulars were used in the original configuration, but they are new in this pull request. How did you manage removing members from VO without them before this change?

Reply:
It is a bug in the older version.